posthog/models/early_access_feature.py

```diff
@@ -30,3 +30,18 @@ def __str__(self) -> str:
         return self.name
 
     __repr__ = sane_repr("id", "name", "team_id", "stage")
+
+    def promote(self) -> None:
+        self.feature_flag.filters = {
+            **self.feature_flag.filters,
+            "super_groups": [
```

Comment:
can we delete the super groups instead to promote? Otherwise any conditions they want to try adding later on wouldn't work?

Reply:
Delete and change release conditions to just a single condition with 100%?